### PR TITLE
WT-9734 Enable stats for csuite tests and format

### PR DESCRIPTION
**Pull request by quchenhao**

### CODE REVIEW

File: test/format/failure_configs/CONFIG.WT-6727 [hunk 1]
```diff
@@ -75,7 +75,6 @@ runs.source=table
 runs.threads=16
 runs.timer=4
 runs.type=row-store
-statistics=0
```
Comment (korteland) [inline test/format/failure_configs/CONFIG.WT-6727:78]: I understand opting configs that previously wanted stats into the `all` setting, but do we want to do the same for configs that wanted stats off? Perhaps it's just people copy-pasting, but given that a number of tests opted out of stats, perhaps it's better to change this at the config level. (i.e. leave the `statistics` config item, have 0 continue to mean off, but change 1 to be `all`.)

**Review by korteland — COMMENTED** [review]
One question about the test/format change, LGTM otherwise.

File: test/format/failure_configs/CONFIG.WT-6727 (hunk 1 above)
Comment (quchenhao) [reply test/format/failure_configs/CONFIG.WT-6727:78]: The change itself is to ensure we always to enable stats for test format. I think we should not run with stat config off so I decided to remove it.

**Comment by quchenhao** [comment]
@etienneptl Can you have a look of this?

**Review by etienneptl — APPROVED** [review]
Sorry I forgot to LGTM after our discussion offline.

**Review by korteland — APPROVED** [review]
Thanks, LGTM! Sorry for the delay.

**Comment by quchenhao** [comment]
evergreen merge

**Pull request merged**